src/bento/dashTemplate.js

```diff
@@ -29,7 +29,7 @@ export const facetSectionVariables = {
 
 export const facetsConfig = [{
   section: PROGRAMS,
-  label: 'FOCUS AREA',
+  label: 'Special Topic',
```

Comment:
I agree with removing the label capitalization. We should also change "CANCER TYPE" label to "Cancer Type" for consistency.